app/pages/project/index.cjsx

```diff
@@ -130,9 +134,9 @@ ProjectPage = React.createClass
         @getSelectedWorkflow(project, @props.preferences)
 
   getSelectedWorkflow: (project, preferences) ->
-    # preference user selected workflow, then project owner set workflow, then default workflow
+    # preference workflow query, then user selected workflow, then project owner set workflow, then default workflow
```

Comment:
I'm not a huge fan of commenting like this as I should be able to read the code to understand this. That said if it's common practice in PFE then....